[2025-03-05T23:04:28.940+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-05T23:04:28.956+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: split_airflow_tasks.merge_csv manual__2025-03-05T23:04:27.891783+00:00 [queued]>
[2025-03-05T23:04:28.973+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: split_airflow_tasks.merge_csv manual__2025-03-05T23:04:27.891783+00:00 [queued]>
[2025-03-05T23:04:28.974+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-03-05T23:04:28.988+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): merge_csv> on 2025-03-05 23:04:27.891783+00:00
[2025-03-05T23:04:28.998+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=46870) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-05T23:04:29.000+0000] {standard_task_runner.py:72} INFO - Started process 46871 to run task
[2025-03-05T23:04:29.001+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'split_***_tasks', 'merge_csv', 'manual__2025-03-05T23:04:27.891783+00:00', '--job-id', '1448', '--raw', '--subdir', 'DAGS_FOLDER/dummyDag.py', '--cfg-path', '/tmp/tmpzn03tmqv']
[2025-03-05T23:04:29.003+0000] {standard_task_runner.py:105} INFO - Job 1448: Subtask merge_csv
[2025-03-05T23:04:29.047+0000] {task_command.py:467} INFO - Running <TaskInstance: split_airflow_tasks.merge_csv manual__2025-03-05T23:04:27.891783+00:00 [running]> on host 629ebb200460
[2025-03-05T23:04:29.166+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='split_***_tasks' AIRFLOW_CTX_TASK_ID='merge_csv' AIRFLOW_CTX_EXECUTION_DATE='2025-03-05T23:04:27.891783+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-05T23:04:27.891783+00:00'
[2025-03-05T23:04:29.168+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-05T23:04:29.203+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-03-05T23:04:29.219+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-05T23:04:29.219+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=split_***_tasks, task_id=merge_csv, run_id=manual__2025-03-05T23:04:27.891783+00:00, execution_date=20250305T230427, start_date=20250305T230428, end_date=20250305T230429
[2025-03-05T23:04:29.265+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-05T23:04:29.302+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-05T23:04:29.304+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
